docs/cli_reference/dfdaemon.md

```diff
@@ -30,6 +30,6 @@ dfdaemon [flags]
 
 ### SEE ALSO
 
-* [dfdaemon gen-doc](dfdaemon_gen-doc.md)	 - Generate Document for dfdaemon command line tool with MarkDown format
-* [dfdaemon version](dfdaemon_version.md)	 - Show the current version of dfdaemon
+* [dfdaemon gen-doc](dfdaemon_gen-doc.md)	 - Generate Document for dfdaemon command line tool with MarkDown format.
```

Comment:
In fact, the files under the `docs/cli_reference` is generated automatically by pouchrobot. 😄